templates/zerver/apps.html

```diff
@@ -86,10 +86,17 @@ <h3 class="center">{% trans %}Ok, I take it back, I'm sorry, please don't go.{%
                             <div class="wrapper">
                                 <h2>MacOS</h2>
                                 <div class="instructions">
-                                    <p class="center">You love your Mac. And you love Zulip. So what could be better than a Zulip app for Mac? Enjoy notifications for messages and PMs in your dock whether you're in Sublime, emacs, or Photoshop.</p>
+                                    <p class="center">
+                                        {% trans %}You love your Mac. And you
+                                        love Zulip. So what could be better
+                                        than a Zulip app for Mac? Enjoy
+                                        notifications for messages and PMs in
+                                        your dock whether you're in Sublime,
+                                        emacs, or Photoshop.{% endtrans %}
+                                    </p>
```

Comment:
Yeah, I agree with that strategy. Until that's fixed, though, I'd rather not create new strings with spurious newlines/indentation -- we shouldn't go and change all the existing screws, but we don't have to make new screws that we don't yet have the right screwdriver for.

For one thing, once we're consistently stripping whitespace in a way we're happy with, if we go back and take strings we've written on one line and reformat them in the source to multiple lines, that should have no effect on the string for translation, so it shouldn't create any work for translators. Whereas if we make the strings multiline in the source now, they'll be multiline for translation, and when we later fix them we'll be sending new, fixed strings to translators.